.github/ISSUE_TEMPLATE/add_dataset.yml

```diff
@@ -0,0 +1,122 @@
+      value: "This dataset is awesome! Here is why..."
+    validations:
+      required: true
+  - type: dropdown
+    id:  dataset-modality
+    attributes:
+      label: Dataset modality
+      description: What is the modality of the data?
+      options:
+        - Text
+        - Image
+        - Audio
+        - Video
+        - Mixed
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: licence
+    attributes:
+      label: Dataset licence
+      description: Please select the licence, if it's not included in the options below add it to the next field
+      options:
+      - Other license
+      - Creative Commons Attribution 1.0 Generic
+      - Creative Commons Attribution 2.0 Generic
+      - Creative Commons Attribution 2.5 Generic
+      - Creative Commons Attribution 3.0 Unported
+      - Creative Commons Attribution 3.0 Austria
+      - Creative Commons Attribution 3.0 United States
+      - Creative Commons Attribution 4.0 International
+      - Creative Commons Attribution Non Commercial 1.0 Generic
+      - Creative Commons Attribution Non Commercial 2.0 Generic
+      - Creative Commons Attribution Non Commercial 2.5 Generic
+      - Creative Commons Attribution Non Commercial 3.0 Unported
+      - Creative Commons Attribution Non Commercial 4.0 International
+      - Creative Commons Attribution Non Commercial No Derivatives 1.0 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 2.0 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 2.5 Generic
+      - Creative Commons Attribution Non Commercial No Derivatives 3.0 Unported
+      - Creative Commons Attribution Non Commercial No Derivatives 3.0 IGO
+      - Creative Commons Attribution Non Commercial No Derivatives 4.0 International
+      - Creative Commons Attribution Non Commercial Share Alike 1.0 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 2.0 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 2.5 Generic
+      - Creative Commons Attribution Non Commercial Share Alike 3.0 Unported
+      - Creative Commons Attribution Non Commercial Share Alike 4.0 International
+      - Creative Commons Attribution No Derivatives 1.0 Generic
+      - Creative Commons Attribution No Derivatives 2.0 Generic
+      - Creative Commons Attribution No Derivatives 2.5 Generic
+      - Creative Commons Attribution No Derivatives 3.0 Unported
+      - Creative Commons Attribution No Derivatives 4.0 International
+      - Creative Commons Attribution Share Alike 1.0 Generic
+      - Creative Commons Attribution Share Alike 2.0 Generic
+      - Creative Commons Attribution Share Alike 2.0 England and Wales
+      - Creative Commons Attribution Share Alike 2.5 Generic
+      - Creative Commons Attribution Share Alike 3.0 Unported
+      - Creative Commons Attribution-Share Alike 3.0 Austria
```

Comment:
The same

Reply:
I've reduced this to the most recent/international version of the Creative Commons licences.  Hopefully, that catches a good chunk of datasets. If we keep getting another licence under the other field we can add this to the dropdown.

Reply:
Great!